src/SFA.DAS.AssessorService.Data/AssessmentOrgsImporter.cs

```diff
@@ -216,7 +216,7 @@ public BlobContainerHelper(IWebConfiguration webConfiguration)
         }
         public async Task<CloudBlobContainer> GetContainer(string containerName)
         {
-            var storageAccount = CloudStorageAccount.Parse(_webConfiguration.IFATemplateStorageConnectionString);     
+            var storageAccount = CloudStorageAccount.DevelopmentStorageAccount;    
```

Comment:
Should this change be pushed? 😸

Reply:
As far as I understand this bit of functionality will only be used to import data to local dev database from a spreadsheet for dev use.